data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1617 @@
+			choice
+				`	"OK, I will explain everything."`
+					goto cognizanceexplanation
+				`	"How can you reach consensus so easily?"`
+			branch cognizanceuntrusted
+				has "remnant untrusted"
+			`	She looks around with an uncertain expression, then nods. "As I am sure has occurred to you, communication by gestures has some disadvantages. We still prefer it for some tasks, but it can be awkward to watch or gesture in the midst of some situations. While we have some vocabulary we can do without needing hands or arms, it is very limited. Fortunately, one of the original abilities of our companions is related to the nervous system. I believe Dawn may have told you a little about it, but was scarce on details." She smiles wryly, then another Remnant takes up the conversation. "To put it simply, we can form ad-hoc neural networks between us, of varying levels of intensity. Without the necessity of speech or gestures, we can have entire conversations in the blink of an eye."`
+			`	He gestures confidently. "A team of Remnant, for instance, could all review incoming data from an experiment, form their own analysis, and then hold what amounts to a conference where each fully presents their thoughts and ideas with supporting data and reasoning within the span of a few minutes." His gestures take on a cautionary tone. "Of course, accelerating to that level takes effort, and it is a long process to learn to manage our thoughts in such a way as to have productive conversations. But the results speak for themselves." He gestures broadly at the group. "One such benefit is that in a policy debate, each person can be heard in depth, and include checking the veracity of data and sources. This has significant benefits for minimizing the numbers of decisions made with incomplete or erroneous data, and maximizes the amount of information available to any one of us."`
+			action
+				set "Remnant: Cognizance Neural Net Explanation"
+			`	A third Remnant steps up. "As to the matter at hand, this means that the sooner any of us hear your report, the sooner everyone will have heard it." They look at you expectantly.`
+				goto cognizanceexplanation
+			label cognizanceuntrusted
+			`	The Remnant smiles mysteriously. "A society has to have some secrets, does it not? Let us just say that our communication networks are very effective. At this point, just know that the sooner you give the report, the sooner that everyone will be able to work on this information." They pause, then continue. "And that does, of course, includes the prefects."`
+			label cognizanceexplanation
+			choice
+				`	"OK, I will explain what happened."`
+				`	"Alright, you convinced me."`
```

Comment:
```suggestion
			choice
				`	"OK, I will explain what happened."`
				`	"Alright, you convinced me."`
			label cognizanceexplanation
```
Jumping to the explanation will cause those two choices to pop up, so the label should be on the other side.